sps30-i2c/sps30.c

```diff
@@ -48,6 +48,8 @@
 #define SPS_CMD_DELAY_USEC 10000
 #define SPS_WRITE_DELAY_USEC 15000
 
+#define SPS30_SERIAL_NUM_WORDS ((SPS30_MAX_SERIAL_LEN) / 2)
```

Comment:
`SENSIRION_NUM_WORDS(SPS30_MAX_SERIAL_LEN)`

Reply:
Ah makes sense. Do you create a PR?

Reply:
https://github.com/Sensirion/embedded-sps/pull/49